packages/gatsby/src/utils/babel-config.js

```diff
@@ -134,55 +57,45 @@ module.exports = async function babelConfig(program, stage) {
 
   // If user doesn't have a custom babelrc, add defaults.
   if (!babelrc) {
-    babelrc = {}
-  }
-  if (!babelrc.plugins) {
-    babelrc.plugins = []
-  }
-  if (!babelrc.presets) {
-    babelrc.presets = []
+    babelrc = {
```

Comment:
instead of what we were doing if there is no babelrc provide a full default config to use that covers most cases. otherwise let the user pick. no magically borrowing of what gatsby has installed in keeping with the other resolution changes we've been making

Reply:
It looks like this may be bugged. `babelrc` is initialized as
```
let babelrc = ... || {};
```
so it should never be falsy?